.github/styles/Vocab/Docs/accept.txt

```diff
@@ -116,3 +116,4 @@ Workbench
 Zabbix
 Zipkin
 ZooKeeper
+overcommit
```

Comment:
We don't really use overcommit as a word, I think we don't need the word in accept.txt and it should still run without errors. Could you check?